errtracker/errtracker.go

```diff
@@ -64,10 +70,19 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		return "", nil
 	}
 
-	machineID, err := ioutil.ReadFile(machineID)
-	if err != nil {
-		return "", err
+	var machineID []byte
+	var err error
+	for _, id := range machineIDs {
```

Comment:
nitpick: I think this would be slightly nicer if extracted into a helper like `func readMachineID() ([]byte, error)`.